src/domain/occurrence/occurrenceGroupInfo/OccurrenceGroupInfo.tsx

```diff
@@ -10,7 +10,8 @@ interface Props {
 const OccurrenceGroupInfo: React.FC<Props> = ({ occurrence }) => {
   const { t } = useTranslation();
 
-  const languages = occurrence.languages.map((language) => language.id);
+  const languages =
```

Comment:
According to type `|| []` seems to be redundant:

```
  const languages = occurrence.languages.edges.map(
    (edge) => edge?.node?.id ?? ''
  );
```

Reply:
Fixed as suggested.